Documentation/GeneralConventions/DirectoryFilenames.rst

```diff
@@ -37,23 +33,32 @@ For alternatives, please look at :ref:`supported-filenames-and-formats`.
 
 .. important::
 
-   The files :file:`Settings.cfg`, :file:`Includes.txt` and
-   :file:`Index.rst` are required in the :file:`Documentation` folder (at least
-   if you are creating an entire sphinx project and not a single-file solution).
+   The files :file:`Settings.cfg` and :file:`Index.rst` are required in the
+   :file:`Documentation` folder (at least if you are creating an entire sphinx
+   project and not a single-file solution).
 
-   The file :file:`composer.json` is mandatory.
+   The file :file:`composer.json` is mandatory inside the project root folder,
+   one level above :file:`Documentation`.
 
 The rest of the files listed here are optional, but strongly recommended.
 
-
 .. _composer-json:
 
 composer.json
 =============
 
-The `composer.json` file must be valid and contain `typo3/cms-core` in the `required` section.
+The :file:`composer.json` file must be valid and contain TYPO3 CMS as
+requirement in the. In order to validate the :file:`composer.json`, the
```

Comment:
"requirement in the." Something missing?  I think it is best to remove "in the". I can push another commit. I looked at the rest. Looks good. :+1: